db/migrate/20220502215806_change_default_state_for_user.rb

```diff
@@ -0,0 +1,5 @@
+class ChangeDefaultStateForUser < ActiveRecord::Migration[5.2]
+  def change
+    change_column_default(:users, :state, :unverified)
```

Comment:
What's up with this change? `"needs_profile"` looks like it is the starting state, is it not? 

```ruby
class User < ApplicationRecord

  VALID_STATES = [
    TEMP = 'temp', # deprecated but still could exist for old accounts
    NEW_SOCIAL = 'new_social',
    UNCLAIMED = 'unclaimed',
    NEEDS_PROFILE = 'needs_profile', # has yet to fill out their user info
    ACTIVATED = 'activated', # means their user info is in place and the email is verified
    UNVERIFIED = 'unverified', # means their user info is in place but the email is not yet verified
  ].freeze
```

Should there be other changes that come with this?

Reply:
I'm basing this change on this comment: https://github.com/openstax/accounts/blob/a6e9cae6eb24c506674a574dff93d00dca593d84/app/routines/ensure_unverified_user.rb#L1

Reply:
wfm